### DISCUSSION TITLE
POST request return 409 and 404

### DISCUSSION THREAD

**Discussion by hitynsun** [Kong/kong#7861] [Help] [answered]
kong version: 0.14.x, but still using [API object route ](https://docs.konghq.com/gateway-oss/0.12.x/admin-api/#add-api)to add apis and plugins 

In my program, after creating apis, I will create plugins immediately. But when calling /apis to create routes, somehow I will get 409, but in the follow-up request to create plugins, will have a 404

We observed this issue a few times on the product environment, but the reason is not clear, and don't know under which condition it will happen.

Appreciate if anyone has an idea, thank you in advance.

**Comment by gszr** [comment, marked as answer]
Hey @hitynsun,

That is an old Kong version, which we no longer support. Is it possible for you to upgrade? Feel free to ask any questions you may have on the upgrade process.

Cheers! 🦍

**Reply by hitynsun** [reply]
Hi @gszr thanks for the reply.

yes, it's an old version as part of our old packages, and currently, there is no plan to upgrade this. 
Apart from upgrading, did you observe similar issues on the old versions?

BTW, if upgrade, to do it seamlessly, which version will you suggest upgrading to? 

Thank you!
